.eslintrc.json

```diff
@@ -16,7 +16,7 @@
     "no-restricted-syntax": [
       2,
       {
-        "selector": "MemberExpression[property.name='toString']",
+        "selector": "CallExpression[callee.property.name='toString']:not(:matches([arguments.0]))",
```

Comment:
before, `.toString` was blacklisted, now `.toString()` without arguments is blacklisted.